example/ios/AirMapsExplorer/AppDelegate.m

```diff
@@ -11,12 +11,16 @@
 
 #import "RCTRootView.h"
 
+#import <GoogleMaps/GoogleMaps.h>
+
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
   NSURL *jsCodeLocation;
 
+  [GMSServices provideAPIKey:@"AIzaSyAeHIC4IG7XKT2Ls5Ti_YZV-6DHQk6dVHE"];
```

Comment:
@spikebrehm that key didn't work because it's not configured to allow this iOS app